src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
+    final ConciseSet conciseSet = new ConciseSet();
+    conciseSet.add(value);
+    final ImmutableConciseSet immutableConciseSet = ImmutableConciseSet.newImmutableFromMutable(conciseSet);
+    return ImmutableConciseSet.intersection(this.bitmap, immutableConciseSet).size() != 0;
```

Comment:
@drcrallen  Proceeding by intersection, like you are doing here is not unreasonable. You are effectively running through the compressed words. Your running time should depend on the compressed size in words.